src/styles/_variables.scss

```diff
@@ -10,41 +10,41 @@ $font-monospace: ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas,
 
 //-- Common Colors --//
 
-$color-dark: black;
-$color-light: white;
-$color-accent-blue: #00bcff;
-$color-primary: $color-accent-blue;
-$color-primary-2: #107da5;
+$color-dark: black; //black-900
```

Comment:
I'm just thinking why we can't just use the variables already defined there. eg `$color-dark: $black-900`? rather than using the comment to note the colour names?

Reply:
This was still work in progress with comments, now everything is clear.